### ISSUE TITLE
comma-spacing with array destructuring assignment

### ISSUE THREAD

**Issue by Kirill89** [eslint/eslint#9968]
In the following code:

```javascript
let [insertId,] = [1, 2];
```

With comma-spacing rule I have an error **A space is required after ','**.

eslint version 4.17.0

Probably should be a way to disable rule for a last comma.

Does it make sense?

[demo](https://eslint.org/demo/#eyJ0ZXh0IjoibGV0IFtpbnNlcnRJZCxdID0gWzEsIDJdOyIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6OCwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6eyJjb21tYS1zcGFjaW5nIjoyfSwiZW52Ijp7fX19)

**Comment by platinumazure** [comment]
Seems reasonable to me. Looks like a similar issue (#3392) was reported for the *leading* comma and fixed in a pull request some time ago, but I don't know if we did the same for the *trailing* comma. I think that would be the correct fix here.

**Comment by not-an-aardvark** [comment]
Is there currently a conflict with `array-bracket-spacing`?

**Comment by Kirill89** [comment]
@not-an-aardvark looks like yes. Because related [issue](https://github.com/eslint/eslint/issues/9760) still open.

**Comment by nzakas** [comment]
Unfortunately, it looks like there wasn't enough interest from the team or community to implement this change. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to be implemented after 90 days tend to never be implemented, and as such, we [close those issues](https://eslint.org/docs/maintainer-guide/issues#when-to-close-an-issue). This doesn't mean the idea isn't interesting or useful, just that it's not something the team can commit to.
